.pylintrc

```diff
@@ -1,2 +1,2 @@
 [good-names]
-disable = invalid-name, too-few-public-methods, eval-used, no-member, duplicate-code, redefined-outer-name
+disable = invalid-name, too-few-public-methods, eval-used, no-member, duplicate-code, redefined-outer-name, unused-argument, unused-variable
```

Comment:
it is good to have those checks `unused-argument, unused-variable`, what is more you do not need to use `.pylintrc` file, because we already have `pylint.yaml` - everything should be there.
Also it is related to `.ruffignore` file